project/AkkaBuild.scala

```diff
@@ -357,8 +348,7 @@ object AkkaBuild extends Build {
   val dontPublishSettings = Seq(
     publishSigned := (),
     publish := (),
-    publishArtifact in Compile := false,
```

Comment:
Pretty sure it was added it was added to filter it out from the zip before, I'll double check.